src/include/migraphx/match/gelu_erf.hpp

```diff
@@ -38,11 +38,10 @@ struct gelu_erf_matcher
     F f;
     auto erf_fn() const
     {
-        return f("erf")(
-            used_once(),
-            arg(0)(used_once(),
-                   f("mul")(either_arg(0, 1)(none_of(has_value(M_SQRT1_2, 1e-3)).bind("x"),
-                                             has_value(M_SQRT1_2, 1e-3)))));
+        return f("erf")(used_once(),
+                        arg(0)(used_once(),
+                               f("div")(either_arg(0, 1)(none_of(has_value(1.414f, 1e-3)).bind("x"),
+                                                         has_value(1.414f, 1e-3)))));
```

Comment:
Use `std::sqrt(2.0)` instead of a hardcoded value. You could probably assign it to a local variable first if you wanted.